projects/phone/src/lib/masks/phone/processors/cut-init-country-code-preprocessor.ts

```diff
@@ -14,6 +14,7 @@ export function cutInitCountryCodePreprocessor({
     metadata: MetadataJson;
 }): MaskitoPreprocessor {
     let isInitializationPhase = true;
+    const code = `+${getCountryCallingCode(countryIsoCode, metadata)} `;
```

Comment:
Do we need to handle exceptions and add a test for unknown country code?

<img width="764" height="349" alt="Image" src="https://github.com/user-attachments/assets/f185c6d8-6a70-4080-936b-e764f87d4edb" />